static/app/components/u2f/u2finterface.tsx

```diff
@@ -49,6 +53,7 @@ class U2fInterface extends Component<Props, State> {
     hasBeenTapped: false,
     deviceFailure: null,
     responseElement: null,
+    authStateElement: null,
```

Comment:
```suggestion
    authStateElement,
```
i'd recommend against setting it as null by default, and instead utilize undefined. `null` is a better representation for when things are in a null state (vs a state yet to be defined)

Reply:
This line complains if I'm missing null because `ref` is `HTMLInputElement | null`.
```typescript
  bindAuthStateElement: React.RefCallback<HTMLInputElement> = ref =>
    this.setState({authStateElement: ref});
```
I think best bet is to leave the default null off, and change the type to be undefined or null